viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/search/SearchPanel.java

```diff
@@ -70,7 +67,7 @@
  */
 @SuppressWarnings("serial")
 public class SearchPanel extends JPanel implements ActionListener, MutableDocument,
-        TreeSelectionListener, DocumentListener {
+        TreeSelectionListener, DocumentListener, BaseSearchComponent {
```

Comment:
To my comments about renaming 'BaseSearchComponent'.  This where I find the use of *Component a bit misleading.